calibrations/mbd/autogen.sh

```
@@ -0,0 +1,8 @@
+#!/bin/sh
+srcdir=`dirname $0`
```

Shellcheck (0.11.0)

[style] Use $(...) notation instead of legacy backticks `...`.

(SC2006)